registries/minikube/configure.sh

```diff
@@ -5,7 +5,8 @@ sudo su -c "echo '{ \"insecure-registries\" : [ \"registry.kube-system.svc.clust
 sudo systemctl daemon-reload
 sudo systemctl restart docker
 
-IMAGE_REPOSITORY_PREFIX="registry.kube-system.svc.cluster.local/u"
+IMAGE_REPOSITORY_HOST="registry.kube-system.svc.cluster.local"
+IMAGE_REPOSITORY_PREFIX="$IMAGE_REPOSITORY_HOST/u"
```

Comment:
The `/u` is a turd left over from when `riff namespace init` had a `--registry-user` flag that could not be empty. We don't need it anymore. I'll open an issue to remove it from other registries.